### ISSUE TITLE
IBM_Community_Grid on Web Nodes creating extreme ansible code lags.

### ISSUE THREAD

**Issue by VDI-Tech-Guy** [ansible/workshops#1032]
##### SUMMARY
When Running the F5 Advanced Workshops lab one of the labs installs Docker/Juiceshop on the Web Nodes as a way to show off more of a real world website rather than the default webpages. 

The installation when IBM_Community_Grid is set to true (defaults) the code will take >20 sometimes 40 minutes to execute.  when the IBM_Community_Grid is set to false the code will run in < 2 minutes 

The understanding is that IBM_Community_Grid should have let the processes in the code run as priority but when looking at the code running on the Web Nodes it seems that IBM_Community_Grid consumes the entire CPU creating a bottleneck where even the web servers are so constraint they become unresponsive (no ssh, no services) for a long time. 

The Code Eventually Completes however the timeframe in which the code should complete is massively greater than when the ibm_community_grid is disabled.

##### ISSUE TYPE

 - Bug Report

##### EXTRA VARS FILE

```
ec2_region: us-west-2
ec2_name_prefix: Test-Lab-MMABIS
student_total: 1
workshop_type: f5
admin_password: Ansible123!
doubleup: no
## Optional Variables
dns_type: aws
workshop_dns_zone: "f5demo.net"
create_login_page: true
autolicense: false
towerinstall: false
```
for more information on the extra vars file please refer to: https://github.com/network-automation/linklight/blob/master/provisioner/README.md

##### ANSIBLE VERSION

```
[root@Elysium git]# ansible --version
ansible 2.9.9
  config file = /root/.ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.8 (default, Apr  2 2020, 13:34:55) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
```

##### CONFIGURATION

```
DEFAULT_HOST_LIST(/root/.ansible.cfg) = ['/root/networking-workshop/lab_inventor
DEFAULT_TIMEOUT(/root/.ansible.cfg) = 60
HOST_KEY_CHECKING(/root/.ansible.cfg) = False
```

##### OS / ENVIRONMENT

CentOS Linux release 7.8.2003 (Core)

##### TOWER

NO Tower

##### Code

This is the code that was run from https://github.com/f5devcentral/f5-bd-ansible-usecases/tree/master/06-F5-WAF-Policy-Management-JuiceShop-Roles

Run the following yaml to execute the code with ibm_community_grid = false  will take <2 minutes to complete.
Run the following yaml to execute the code with ibm_community_grid = true (or not defined which is enabled) will take > 20 minutes to complete. 
```
ansible-playbook JuiceShop-Docker-Deploy.yaml
```

Biggest hangups seem to be around installing Docker/NGINX and Installing GPG Keys with (Yum Functions)

**Comment by VDI-Tech-Guy** [comment]
Please note i do understand you can disable the community grid other ways, this was just to see if theres a way to make the grid work with the code so that the code runs under a respectable amount of time rather than disabling it for labs like RHPDS

**Comment by IPvSean** [comment]
hey @VDI-Tech-Guy when you provision your workshops do you see this Job Template in Ansible Tower->
![image](https://user-images.githubusercontent.com/6537680/99849813-34e61400-2b4a-11eb-9b20-bd7943bcd269.png)

**Comment by VDI-Tech-Guy** [comment]
@IPvSean  - Yes i do see that option and the yaml file in the F5-workshop folder, if this is the path we need to add to our lab i am fine with that and we can close this out.

**Comment by IPvSean** [comment]
@VDI-Tech-Guy testing fix with PR #1049
